middle_end/flambda2/terms/effects_and_coeffects.ml

```diff
@@ -12,21 +12,28 @@
 (*                                                                        *)
 (**************************************************************************)
 
-type t = Effects.t * Coeffects.t
+type t = Effects.t * Coeffects.t * Placement.t
 
-let [@ocamlformat "disable"] print fmt (eff, coeff) =
-  Format.fprintf fmt "%a * %a" Effects.print eff Coeffects.print coeff
+let [@ocamlformat "disable"] print fmt (eff, coeff, dup) =
```

Comment:
Please remove the disable attribute from this function + reformat

Reply:
done in dbe72df5b8ad57ad5a18780e46e78ad7a523c4ab